docs/iot/debugging.md

```diff
@@ -30,11 +34,14 @@ Debugging .NET on Raspberry Pi from Visual Studio Code requires configuration st
 
 ### Enable SSH on the Raspberry Pi
 
-SSH is required for remote debugging. To enable SSH, [refer to *Enable SSH* in the Raspberry Pi documentation](https://www.raspberrypi.com/documentation/computers/remote-access.html#setting-up-an-ssh-server).
+SSH is required for remote debugging. To enable SSH, [refer to *Enable SSH* in the Raspberry Pi documentation](https://www.raspberrypi.com/documentation/computers/remote-access.html#setting-up-an-ssh-server). Ensure that you have configured [passwordless SSH](https://www.raspberrypi.com/documentation/computers/remote-access.html#passwordless-ssh-access).
+
+> [!IMPORTANT]
+> This example requires you to configure [passwordless SSH](https://www.raspberrypi.com/documentation/computers/remote-access.html#passwordless-ssh-access) on your Raspberry Pi, as OpenSSH doesn't support passing passwords on the command line. If you need to use a password, consider substituting the *plink* tool included with [PuTTY](https://www.putty.org/) for *ssh*.
 
 ### Install the Visual Studio Remote Debugger on the Raspberry Pi
 
-Within a Bash console on the Raspberry Pi (either locally or via SSH), execute the following command. This command downloads and installs the Visual Studio Remote Debugger on the Raspberry Pi:
+Within a Bash console on the Raspberry Pi (either in a local session or via SSH), execute the following command. This command downloads and installs the Visual Studio Remote Debugger on the Raspberry Pi:
```

Comment:
We should always use "run" instead of "execute" for translation purposes. "execute" can be thought of as killing someone.

```suggestion
Within a Bash console on the Raspberry Pi (either in a local session or via SSH), run the following command. This command downloads and installs the Visual Studio Remote Debugger on the Raspberry Pi:
```